packages/node-experimental/src/sdk/init.ts

```diff
@@ -0,0 +1,38 @@
+import { hasTracingEnabled } from '@sentry/core';
+import { defaultIntegrations as defaultNodeIntegrations, init as initNode } from '@sentry/node';
+
+import { getAutoPerformanceIntegrations } from '../integrations/getAutoPerformanceIntegrations';
+import { Http } from '../integrations/http';
+import type { NodePreviewOptions } from '../types';
+import { NodePreviewClient } from './client';
+import { initOtel } from './initOtel';
+
+const ignoredDefaultIntegrations = ['Http', 'Undici'];
+
+export const defaultIntegrations = [
+  ...defaultNodeIntegrations.filter(i => !ignoredDefaultIntegrations.includes(i.name)),
+  new Http(),
+];
+
+/**
+ * Initialize Sentry for Node.
+ */
+export function init(options: NodePreviewOptions | undefined = {}): void {
```

Comment:
m: We probably should set the sdk metadata for this package, right?

Reply:
good point 👍